src/scripts/containers/widgets/modals/ModalContainer.js

```diff
@@ -1,13 +1,14 @@
 /* @flow */
 import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux'
-import { getModalContent, getModalStatus } from 'selectors/index'
+import { getModalName, getModalProps, getModalStatus } from 'selectors/index'
 import { closeModal } from 'actions/ModalActions'
 import Modal from 'components/widgets/modals/Modal'
 import type { RootState } from 'types/ApplicationTypes'
 
 const mapStateToProps = (state: RootState) => ({
-  modalContent: getModalContent(state),
+  modalName: getModalName(state),
```

Comment:
Changed to `modalName` as it is a better representation of what this property is